input/about/bylaws.md

```diff
@@ -67,7 +67,17 @@ Amended and Restated Effective June 4, 2020
 
 **(e)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Limited Special Right for Director Appointed by Founding Member.** In connection with any vote to materially change the Foundation’s Membership Policy, Director Election Policy, Project Governance Policy, or any Intellectual Property-related agreements or policies, (a) a “no” vote by the director appointed by the Founding Member will result in the disapproval of a proposed action, regardless of the number of votes for approval), and (b) such director must be present as part of any quorum (i.e. if that director is not present, the Board will not have a quorum for the matter, regardless of the number of other directors present).
 
-**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Notices.** All notices required under this Article III will be given to all directors in office at the time of such notice and may be given by email at least 48 hours in advance of the meeting or by first class mail to such director’s last known business address at least three business days in advance.
+**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Member Review.** Prior to voting on any amendments to the Foundation’s Bylaws, Articles of Incorporation or the Foundation's Membership Policy, the Director Election Policy, the Project Governance Policy, or any of the Foundation’s Intellectual Property-related agreements or policies, the Board shall use reasonable efforts to preview the proposed amendments with the Foundation’s Members, using the following process:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(i)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;post the proposed amendments as “pull requests” on the Foundation’s Github, or through such other methodology determined by the Board that is accessible to Members for review and feedback;
```

Comment:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(i)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;post the proposed amendments as “pull requests” within the Foundation’s GitHub organization, or through such other methodology determined by the Board that is accessible to Members for review and feedback;
```

(This was mostly to fix the capitalization of GitHub, but we also treating "GitHub" not as a noun that the Foundation could have one of.)